packages/eas-cli/src/credentials/manager/ManageIosBeta.ts

```diff
@@ -0,0 +1,125 @@
+    if (iosAppBuildCredentialsArray.length === 0) {
+      return null;
+    }
+
+    // The order in which we choose the distribution type from least to most preferred
+    const typePriority = [
+      IosDistributionType.Development,
+      IosDistributionType.AdHoc,
+      IosDistributionType.Enterprise,
+      IosDistributionType.AppStore,
+    ];
+    const sortedIosAppBuildCredentialsArray = iosAppBuildCredentialsArray.sort(
+      (buildCredentialsA, buildCredentialsB) =>
+        typePriority.indexOf(buildCredentialsA.iosDistributionType) -
+        typePriority.indexOf(buildCredentialsB.iosDistributionType)
+    );
+    return sortedIosAppBuildCredentialsArray[0].iosDistributionType;
+  }
+
+  async runAsync(manager: CredentialsManager, ctx: Context): Promise<void> {
+    // TODO uncomment this before committing!
+    //manager.pushNextAction(this);
+    await ctx.bestEffortAppStoreAuthenticateAsync();
+
+    const accountName = ctx.hasProjectContext
+      ? getProjectAccountName(ctx.exp, ctx.user)
+      : ensureActorHasUsername(ctx.user);
+
+    const account = findAccountByName(ctx.user.accounts, accountName);
+    if (!account) {
+      throw new Error(`You do not have access to account: ${accountName}`);
+    }
+    if (ctx.hasProjectContext) {
+      const appLookupParams = this.getAppLookupParamsFromContext(ctx);
+      const iosAppCredentials = await ctx.newIos.getIosAppCredentialsWithBuildCredentialsAsync(
+        appLookupParams
+      );
+      if (!iosAppCredentials) {
+        displayEmptyIosCredentials(appLookupParams);
+      } else {
+        this._currentDistributionType = this.chooseDistributionType(iosAppCredentials);
+        displayAllIosCredentials(iosAppCredentials, this._currentDistributionType);
+      }
+    }
+
+    const projectSpecificActions: { value: ActionType; title: string }[] = ctx.hasProjectContext
+      ? []
+      : [];
+
+    const { action } = await promptAsync({
+      type: 'select',
+      name: 'action',
+      message: 'What do you want to do?',
+      choices: [
+        ...projectSpecificActions,
+        {
+          value: ActionType.RemoveDistributionCertificate,
+          title: 'Remove Distribution Certificate',
+        },
+      ],
+    });
+
+    manager.pushNextAction(new PressAnyKeyToContinue());
+    manager.pushNextAction(this.getAction(ctx, accountName, action));
+  }
+
+  private getAppLookupParamsFromContext(ctx: Context): AppLookupParams {
+    const projectName = ctx.exp.slug;
+    const accountName = getProjectAccountName(ctx.exp, ctx.user);
+
+    const account = findAccountByName(ctx.user.accounts, accountName);
+    if (!account) {
+      throw new Error(`You do not have access to account: ${accountName}`);
+    }
+
+    const bundleIdentifier = ctx.exp.ios?.bundleIdentifier;
+    if (!bundleIdentifier) {
+      throw new Error(`ios.bundleIdentifier needs to be defined in your eas.json file`);
```

Comment:
```suggestion
      throw new Error(`ios.bundleIdentifier needs to be defined in your app.json file`);
```

Reply:
Or use https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/project/projectUtils.ts#L112
to get an actual name of the config file or files